tests/runmode-test/test.yaml

```diff
@@ -0,0 +1,13 @@
+requires:
+  min-version: 7.0.3 
+  pcap: false
+
+exit-code: 1
+
+args: 
```

Comment:
nit: While reviewing locally, I've noticed some trailing whitespace at the end of `min-version` and `args:` lines. Could you remove those, please?